transformer_engine/pytorch/module/base.py

```diff
@@ -151,7 +152,7 @@ def add_ub(
         num_splits: int = 4,
         aggregate: int = 0,
     ) -> None:
-        dtype = torch.uint8 if (use_fp8 and name in fp8_buf) else torch.bfloat16
+        dtype = torch.uint8 if (use_fp8 and name in fp8_buf)
```

Comment:
Do you mean:
```suggestion
        dtype = torch.uint8 if (use_fp8 and name in fp8_buf) else dtype
```

Reply:
Yes, but since `dtype `is assigned as an input parameter, so if we don't use fp8, no need to change it.

Reply:
I think it'll cause a runtime error, unless it's a syntax I'm not familiar with. This change shouldn't affect the intended behavior, so I'll just change it to be more staightforward.